### PR TITLE
Added tests for `ratio.jl`

### PR DESCRIPTION
**Pull request by itsdebartha**
Ratio tests for stratified sampling

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -42,6 +42,11 @@ apiclus2[!, :pw] = fill(757 / 15, (size(apiclus2, 1),)) # Correct api mistake fo
 dclus2 = SurveyDesign(apiclus2; clusters = :dnum, weights = :pw) # Create SurveyDesign
 dclus2_boot = dclus2 |> bootweights # Create replicate design
 
+# Stratified sample
+apistrat = load_data("apistrat") # Load API dataset
+dstrat = SurveyDesign(apistrat; strata = :stype, weights = :pw) # Create SurveyDesign
+dstrat_boot = dstrat |> bootweights # Create replicate design
+
```
Comment (ayushpatnaikgit) [inline test/runtests.jl:49]: This already there in line 22
```suggestion
```

File: test/runtests.jl [hunk 2]
```diff
@@ -20,6 +20,7 @@ bsrs_regex = ReplicateDesign(bsrs.data, REPLICATES_REGEX, weights = :pw)  # usin
 # Stratified sample
 apistrat = load_data("apistrat") # Load API dataset
 dstrat = SurveyDesign(apistrat, strata = :stype, weights = :pw) # Create SurveyDesign
+dstrat_boot = dstrat |> bootweights # Create replicate design
```
Comment (ayushpatnaikgit) [inline test/runtests.jl:23]:
```suggestion
```

Comment (itsdebartha) [reply test/runtests.jl:23]: So we don't need to add replicate weights for stratified sampling?

Comment (smishr) [reply test/runtests.jl]: This is already there `bstrat` in line 24

**Review by smishr — APPROVED** [review]
tests passed

**Pull request merged**